### PR TITLE
Bxmsdoc 3333 master Release notes for BA 7.1.1 patch

### PR DESCRIPTION
**Pull request by emmurphy1**
[BXMSDOC-3333](https://issues.jboss.org/browse/BXMSDOC-3333)
[RHPAM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-3333-pam/#rn-7.1.1-fixed-issues-ref)
[RHDM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-3333-dm/#rn-7.1.1-fixed-issues-ref)

Please review the Fixed issues in 7.1.1 sections.

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ifdef::PAM[]
+== Business Central
+* Cannot open a task form from the Task Inbox in Business Central [https://issues.jboss.org/browse/RHPAM-1566[RHPAM-1566]]
+* Unable to import projects from a secured Git repository [https://issues.jboss.org/browse/RHPAM-1617[RHPAM-1617]]
+* Business Calender does not work as expected when the timer is set in some conditions [https://issues.jboss.org/browse/RHPAM-1589[RHPAM-1589]]
+
+== Decision engine
+//BRE
+* MVEL uses random parameter type methods for `BigDecimal.valueOf(100)` [https://issues.jboss.org/browse/RHDM-699[RHDM-699]]
+* MVEL expression (1 + 2 * 3 + $v )  causes "no such method or function" error for bind variable  [https://issues.jboss.org/browse/RHDM-707[RHDM-707]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc]: causes the or a?

File: doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+== Process engine
+//jBPM Core
+* RHPAM 7.1.0 contains a file handle leak in AbstractUserGroupInfo [https://issues.jboss.org/browse/RHPAM-1586[RHPAM-1586]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc:25]: Is AbstractUserGroupInfo a file or a property?

File: doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+== Decision engine
+//BRE
+* MVEL uses random parameter type methods for `BigDecimal.valueOf(100)` [https://issues.jboss.org/browse/RHDM-699[RHDM-699]]
+* MVEL expression (1 + 2 * 3 + $v )  causes "no such method or function" error for bind variable  [https://issues.jboss.org/browse/RHDM-707[RHDM-707]]
+
+== {PRODUCT} on OpenShift
+* s2i build fails for multi module Maven projects [https://issues.jboss.org/browse/RHDM-765[RHDM-765]]
+
+== Maven repository
+* Maven repository is missing org.kie.server:kie-server-controller-client [https://issues.jboss.org/browse/RHDM-770[RHDM-770]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc]: missing the..?

File: doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+== Decision Central
+* Unable to import projects from a secured Git repository [https://issues.jboss.org/browse/RHPAM-1617[RHPAM-1617]]
+
+== Installation and migration
+* The update tool in Decision Server distributions contains a typo [https://issues.jboss.org/browse/RHDM-758[RHDM-758]]
+
+== Decision engine
+//BRE
+* MVEL uses random parameter type methods for `BigDecimal.valueOf(100)` [https://issues.jboss.org/browse/RHDM-699[RHDM-699]]
+* MVEL expression (1 + 2 * 3 + $v )  causes "no such method or function" error for bind variable  [https://issues.jboss.org/browse/RHDM-707[RHDM-707]]
```
Comment (michelehaglund) [inline doc-content/enterprise-only/release-notes/rn-7.1.1-fixed-issues-ref.adoc:41]: causes a or the..

**Review by MarianMacik — COMMENTED** [review]
2 issues from RH DM should be for RH PAM as well.

**Pull request merged**